cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,52 @@
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+import HomeAndCataloguePageObject from
+  '../support/pages/homeСatalogue.pageObject';
+import BtnPageObject from
+  '../support/pages/buttons.pageObject';
+import CheckoutForm from
+  '../support/pages/checkoutForm.pageObject';
+import faker from 'faker';
 
-  });
+describe('Demoblaze (POM basics)', () => {
+  const homePage = new HomeAndCataloguePageObject();
+  const btn = new BtnPageObject();
+  const checkForm = new CheckoutForm();
+
+  const testData = {
+    name: faker.name.firstName(),
+    country: faker.address.country(),
+    city: faker.address.city(),
+    card: faker.finance.creditCardNumber(),
+    month: faker.random.number({ min: 1, max: 12 }),
+    year: faker.random.number({ min: 2013, max: 2033 }),
+    cartLink: 'Cart',
+    categoryName: 'Laptops',
+    productName: 'Sony vaio i7',
+    message: 'Product added',
+    successMessage: 'Thank you for your purchase!'
+  };
 
-  it('', () => {
+  before(() => {
+    homePage.visit();
+  });
 
+  it('should place an order', () => {
+    homePage.clickOnCategory(testData.categoryName);
+    homePage.clickOnProduct(testData.productName);
+    btn.clickOnAddBtn();
+    homePage.assertAlert(testData.message);
```

Comment:
there is no such method in the default home catalogue, so it means that u changed it) add it pls

Reply:
> there is no such method in the default home catalogue, so it means that u changed it) add it pls

Yes, sorry I was wrong, I didn't add this file to the commit